core/node/rpc/stream_test.go

```diff
@@ -0,0 +1,54 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/river-build/river/core/node/crypto"
+	"github.com/river-build/river/core/node/protocol"
+)
+
+func TestGetStreamEx(t *testing.T) {
+	tt := newServiceTester(
+		t,
+		serviceTesterOpts{
+			numNodes: 1,
+			start:    true,
+			btcParams: &crypto.TestParams{
+				AutoMine:         true,
+				AutoMineInterval: 200 * time.Millisecond,
+				MineOnTx:         true,
+			},
+		},
+	)
+	require := tt.require
+
+	alice := tt.newTestClient(0)
+	_ = alice.createUserStream()
+	spaceId, _ := alice.createSpace()
+	channelId, _ := alice.createChannel(spaceId)
+
+	for count := range 100 {
+		alice.say(channelId, fmt.Sprintf("hello from Alice %d", count))
+	}
+
+	time.Sleep(1 * time.Second)
+
+	stream := alice.getStream(channelId)
+	fmt.Println(len(stream.GetEvents())) // Prints 0
```

Comment:
Use `require.Eventually` and `t.Logf` instead of sleep and println.

Reply:
Yea, this is just for debugging.

Reply:
Yea, this is just for debugging